docs/pbs/index.md

```diff
@@ -329,9 +329,13 @@ The `qinteractive` command provides a convenient way to start an interactive job
 The following example shows how to start an interactive job on either `derecho` or `casper` :
 
 ```bash
-qinteractive
+qinteractive -A <project_code>
 ```
 
+!!! tip
+    The `-A` flag is needed if users have not specified a default project code in their environment. The default project code can be specified using environment variable `DAV_PROJECT` on Casper and `PBS_ACCOUNT` on Derecho.
```

Comment:
What do you think about using the proposed account variable changes here instead? Something like:

The `-A` flag is needed if users have not specified a default project code in their environment. The default project code can be specified using the environment variable `PBS_ACCOUNT`. Default project codes can also be specified for a specific system using `PBS_ACCOUNT_DERECHO` and `PBS_ACCOUNT_CASPER`.

I can make the change to the script when we are ready to publish, if this sounds good to you. I'll leave DAV_PROJECT as a legacy Casper setting, of course.

Reply:
Thanks @vanderwb . I updated the text to what you suggested verbatim. 
Personally, I like using `PBS_ACCOUNT` for both machines and a separate `PBS_ACCOUNT_machine` for specified machine. The only concern is to make sure the behavior works as we agree upon. For example:

If users have set : 
```
PBS_ACCOUNT = account-a
PBS_ACCOUNT_DERECHO = account-b
PBS_ACCOUNT_CASPER = account-c
```
If they submit a job to Derecho, does it use `account-a` or `account-b` (i.e. I think the system specific should overwrite and it should use `account-b`).

Reply:
Yes - I would make the system-specific variable take precedence.